CHANGELOG.txt

```diff
@@ -2,6 +2,13 @@
 MABWiser CHANGELOG
 =====================
 
+January, 24, 2022 2.6.0
+-------------------------------------------------------------------------------
+minor:
```

Comment:
As a complete new functionality (correct me if I am wrong) this seems major to me (as in, if I were  a user, I would wanna learn this)